Gemfile

```diff
@@ -123,6 +123,12 @@ gem 'database_cleaner'
 gem 'redlock', '~> 1.0'
 gem 'httparty', '~> 0.21'
 
+# Sentry-ruby for error handling
+gem "sentry-ruby"
+gem "sentry-rails"
+gem "sentry-sidekiq"
+gem "sentry-resque"
```

Comment:
why resque? we are only using sidekiq here I beleive

Reply:
Thanks I wasn't too sure if it'd be useful to keep handy when reading the description of it: "Sentry's Ruby SDK allows users to report messages, exceptions, and tracing events."